auth0/src/main/java/com/auth0/android/provider/AuthenticationActivity.java

```diff
@@ -12,8 +12,6 @@
 public class AuthenticationActivity extends Activity {
 
     static final String EXTRA_USE_BROWSER = "com.auth0.android.EXTRA_USE_BROWSER";
```

Comment:
Please remove this flag, as there's no longer a decision to be made around what to use

Reply:
👍 removed in a new commit